hooks/build

```diff
@@ -26,44 +27,92 @@ function build_image(){
         # Latest version - we do not need the version argument - latest version is defined in Dockerfile itself
         docker build --rm --build-arg BUILD_DATE="${BUILD_DATE}" \
             --build-arg CHEVERETO_VERSION="master" \
-            -t "${image_full_name}" .
+            -t "${image_full_name}-amd64" .
+        docker build --rm --build-arg BUILD_DATE="${BUILD_DATE}" \
+            --build-arg CHEVERETO_VERSION="master" \
+            -t "${image_full_name}-arm32v7" --platform=linux/arm/v7 -f Dockerfile.arm32v7 .
```

Comment:
Please make the list of architectures we support a list, loop through it and build all our images. This way we can change thing more easily.

Also, I would like to add support for ARMv6. A Pi zero is ARMv6 and thus can't run the image for ARMv7